components/infobox/wikis/halo/infobox_patch_custom.lua

```diff
@@ -0,0 +1,71 @@
+---
+-- @Liquipedia
+-- wiki=halo
+-- page=Module:Infobox/Patch/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Patch = require('Module:Infobox/Patch')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local _args
+
+local _GAME = mw.loadData('Module:GameVersion')
+
+local CustomPatch = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomPatch.run(frame)
+	local customPatch = Patch(frame)
+	_args = customPatch.args
+	customPatch.createWidgetInjector = CustomPatch.createWidgetInjector
+	customPatch.getChronologyData = CustomPatch.getChronologyData
+	customPatch.addToLpdb = CustomPatch.addToLpdb
+	return customPatch:createInfobox(frame)
+end
+
+function CustomPatch:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{
+		name = 'Game Version',
+		content = {CustomPatch._getGameVersion()},
+		options = {makeLink = true}
+	})
+	return widgets
+end
+
+function CustomPatch:addToLpdb(lpdbData)
```

Comment:
commons passes args not lpdbdata
so either let it pass the args
```suggestion
function CustomPatch:addToLpdb(args)
```
and change all the `_args` to `args` in this function
or use
```suggestion
function CustomPatch:addToLpdb()
```